### ISSUE TITLE
Change newsletter sign up box on home page to reflect more than one newsletter

### ISSUE THREAD

**Issue by JoPratt** [akvo/akvo-web#545]
The sign up function in the Let's be friends area of the home page needs to change to link to new Newsletter sign up page. Gabe and Jo discussed options: 

[24/02/2015 13:40:51] Jo Pratt: Do you know what will happen to the newsletter sign up box on teh bottom right of the home page. Could that also be amended to link to this page instead?
[24/02/2015 13:44:50] Gabe Brady: I can definitely switch the front-page box to link to that page too
[24/02/2015 13:48:58] Jo Pratt: It will need changing around a bit on the home page as it doesn't have an icon. Maybe it could replace the Contact Akvo icon - Linda has a newsletter one that would work there - and then lose the subscribe box altogether?
[24/02/2015 13:53:36] Gabe Brady: Sure, it could be an icon in the "let's be friends" area, but it might be a bit less prominent there?
[24/02/2015 14:38:15] Jo Pratt: Good point re prominence. Linda is suggesting having a box a bit like the widget box on the blog sidebar, but narrower

**Comment by charmermark** [comment]
My sense is that having a direct newsletter signup on the home page itself adds a layer of immediate engagement that is reduced by needing to click to another page. Would be interested in Gabe and Loïc's opinion on that.

**Comment by Lynngre** [comment]
@gabemart & @loicsans please add your input re @charmermark 's comment.

**Comment by gabemart** [comment]
I agree with @charmermark - having an immediate sign-up available will reduce friction and probably result in more people signing up to a newsletter.

Is there a central newsletter that we could anticipate the most people would be interested in? If so, I would propose a widget that lets people immediately sign up for the main newsletter, and has a link to the newsletter page with text something like "See all our newsletters and read the archives"

**Comment by Lynngre** [comment]
This requires further consideration.  To discuss in Web team meeting Tuesday 14 April.

**Comment by JoPratt** [comment]
I like Gabe's suggestion above
